tasks/libs/pipeline_notifications.py

```diff
@@ -37,17 +38,49 @@ def get_failed_jobs(project_name, pipeline_id):
         }
         final_failed_jobs.append(final_status)
 
-    pprint(final_failed_jobs)
-
     return final_failed_jobs
 
 
-def find_job_owners(failed_jobs, owners_file=".gitlab/JOBOWNERS"):
+class Test:
+    PACKAGE_PREFIX = "github.com/DataDog/datadog-agent/"
+
+    def __init__(self, owners, name, package):
+        self.name = name
+        self.package = self.__removeprefix(package)
+        self.owners = self.__get_owners(owners, package)
+
+    def __removeprefix(self, package):
+        return package[len(self.PACKAGE_PREFIX) :]
+
+    def __get_owners(self, OWNERS, package):
+        owners = OWNERS.of(self.__removeprefix(package))
+        return [name for (kind, name) in owners if kind == "TEAM"]
+
+
+def read_owners(owners_file):
     from codeowners import CodeOwners
 
     with open(owners_file, 'r') as f:
-        owners = CodeOwners(f.read())
+        return CodeOwners(f.read())
+
 
+def get_failed_tests(project_name, job, owners_file=".gitlab/CODEOWNERS"):
```

Comment:
By default, the `CODEOWNERS` file should be in `.github/`, shouldn't it?
```suggestion
def get_failed_tests(project_name, job, owners_file=".github/CODEOWNERS"):
```

Reply:
woops, fixed